AnkiDroid/src/main/java/com/ichi2/anki/dialogs/tags/TagsDialog.kt

```diff
@@ -269,6 +270,7 @@ class TagsDialog : AnalyticsDialogFragment {
             // utilize the addTagFilter to append '::' properly by appending a space to prefixTag
             inputET.setText("$prefixTag ")
         }
+        setFocusAndOpenKeyboard(inputET) { inputET.setSelection(inputET.text.length) }
```

Comment:
Is `setFocusAndOpenKeyboard()` necessary here? Didn't seem like it when I tested it
```suggestion
       inputET.setSelection(inputET.text.length)
```

Reply:
ok